docker-compose.yml

```diff
@@ -55,10 +55,11 @@ services:
       # Keep in sync with the development key in
       # https://github.com/opengovsg/formsg-javascript-sdk/blob/develop/src/resource/signing-keys.ts
       - SIGNING_SECRET_KEY=HDBXpu+2/gu10bLHpy8HjpN89xbA6boH9GwibPGJA8BOXmB+zOUpxCP33/S5p8vBWlPokC7gLR0ca8urVwfMUQ==
-      - TWILIO_ACCOUNT_SID
-      - TWILIO_API_KEY
-      - TWILIO_API_SECRET
-      - TWILIO_MESSAGING_SERVICE_SID
+      # Mock Twilio credentials. SMSes do not work in dev environment.
```

Comment:
Sad, this means we need to override this on our docker-compose with the appropriate credentials if we want smsing in dev? Increases the risk of secrets being pushed if we accidentally forget too :(

Reply:
tbh I'm not sure there's a good solution which actually allows SMSes in dev. even with twilio test credentials, the SMSes don't actually get sent.